core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -549,16 +573,22 @@ public void dayOfYearWithUnderscoresLeapYear() {
   }
 
   public void testInvalidDayOfYear(String date) {
```

Comment:
`@Test`

Reply:
Function renamed instead in 85c93e677ba64de401fa4b4115acfe428005e003 as it is a helper function